usdigitalresponse/usdr-gost
# feat: remove "Occasional Updates" toggle from profile page

### Ticket #2317
## Description
I removed the toggle from the "My Profile" page (see screenshot). I didn't delete any of the other code added to support "Occasional Updates" because my impression is that we want to restore this feature once we get email lists working.

I also fixed an issue I discovered during testing - toggling any of email settings on "My Profile" was causing a 403 error in the background. I don't think this was affecting the user experience but it could cause noise in Datadog. See in-code comments for details.

## Screenshots / Demo Video
<img width="1319" alt="Screenshot 2023-12-20 at 3 06 50 PM" src="https://github.com/usdigitalresponse/usdr-gost/assets/8216592/5fa80718-186c-4717-a26b-d17bb2160b44">


## Testing
Load the app and check there is no toggle on the profile page. No relevant feature flags.

I manually tested the following and saw no errors logged in the browser console:
- Load the "My Profile" page
- Toggle on/off the remaining 2 toggles (New Grant Digest, Grants Assignment)

### Automated and Unit Tests

### Manual tests for Reviewer

## Checklist
